### PR TITLE
Add compat data for PushEvent

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/PushEvent

### CODE REVIEW

File: api/PushEvent.json [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+            }
+          ],
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/PushEvent.json]: Both versions of Opera should be 37.

File: api/PushEvent.json [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/PushEvent.json]: 37 again.

File: api/PushEvent.json [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+      },
+      "data": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PushEvent/data",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
```
Comment (jpmedley) [inline api/PushEvent.json]: 57

File: api/PushEvent.json [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/PushEvent.json]: 57

File: api/PushEvent.json [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+              {
+                "version_added": "48",
+                "notes": "Push enabled by default."
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```
Comment (jpmedley) [inline api/PushEvent.json]: 44

File: api/PushEvent.json [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/PushEvent.json]: 44

**Comment by bunnybooboo** [comment]
Changes made due to researching via #1010

**Pull request merged**